daffodil-io/src/main/scala/org/apache/daffodil/io/StringDataInputStreamForUnparse.scala

```diff
@@ -84,5 +84,4 @@ final class StringDataInputStreamForUnparse
   override def skip(nBits: Long, finfo: FormatInfo): Boolean = doNotUse
   override def resetBitLimit0b(savedBitLimit0b: MaybeULong): Unit = doNotUse
   // $COVERAGE-ON$
-  override def validateFinalStreamState: Unit = {} // does nothing
```

Comment:
If this removal is valid. also remove the `// $COVERAGE-ON$`.

Reply:
dead code. Will fix.